test/include/seqan3/test/pretty_printing.hpp

```diff
@@ -0,0 +1,32 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \brief Contains an hacky overload for the googletest PrintTo function.
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ */
+
+#pragma once
+
+
+namespace seqan3
+{
+
+//!\cond DEV
+//!\brief Hacky overload for the googletest PrintTo function that always delegates to our debug_stream.
+template <typename t>
+    requires true
+void PrintTo (t const & v, std::ostream * out)
+{
+    std::ostringstream o;
+    debug_stream_type my_stream{o};
+    my_stream << v;
+    o.flush();
+    *out << o.str();
+}
+//!\endcond
+}
```

Comment:
add `// namespace seqan3` (seems to me like a convention in our code for closing namespaces)